.vscode/settings.json

```diff
@@ -1,4 +1,8 @@
 {
+  "sonarlint.connectedMode.project": {
```

Comment:
Out of interest, what does this do?

Reply:
It's the setup for the SonarQube vscode extension